Tests/L0/SonarQubePostTest/PRCA/ReportProcessorTests.ps1

```diff
@@ -1,9 +1,10 @@
 [CmdletBinding()]
 param()
 
+Import-Module -Name "$PSScriptRoot\..\..\..\..\Tasks\SonarQubePostTest\PRCA\ReportProcessor-Module.psm1" -Verbose
+Import-Module -Name "$PSScriptRoot\..\..\..\..\Tasks\SonarQubePostTest\PRCA\PostComments-Module.psm1" -Verbose
```

Comment:
I have already transformed this script to a module based on feedback on an earlier PR